staging/src/k8s.io/cloud-provider/app/testing/testserver.go

```diff
@@ -73,6 +73,16 @@ type TestServer struct {
 // files that because Golang testing's call to os.Exit will not give a stop channel go routine
 // enough time to remove temporary files.
 func StartTestServer(ctx context.Context, customFlags []string) (result TestServer, err error) {
+	return StartTestServerWithOptions(ctx, customFlags, app.DefaultInitFuncConstructors, names.CCMControllerAliases())
```

Comment:
The same `app.DefaultInitFuncConstructors` and `names.CCMControllerAliases()` are being enumerated here now instead of in the bowels of the `StartTestServer`.  `StartTestServer` runs exactly the same. There is now a new `StartTestServerWithOptions()` to allow others to specify these default init constructors and CCM controllers.